src/components/web3/StartCall.tsx

```diff
@@ -170,11 +183,18 @@ export const StartCall: React.FC<Props> = ({
             <OptionalSettings
               startCall={true}
               web3Account={web3Account}
+              web3Address={web3Address}
               permissionType={permissionType}
               nfts={nfts}
               poaps={poaps}
               nftCollections={nftCollections}
               nft={nft}
+              exceptionList={exceptionList}
+              setExceptionList={setExceptionList}
+              allowList={allowList}
+              setAllowList={setAllowList}
+              isExceptionAddressWrong={isExceptionAddressWrong}
+              setIsExceptionAddressWrong={setIsExceptionAddressWrong}
               setPermissionType={setPermissionType}
               setNft={setNft}
```

Comment:
There's now a _lot_ of params here. This may be not for this PR but I think we need to review how we're managing state as this is at the point of becoming an unmanageable mess.

Especially since we repeat it twice.